examples/docker/docker-compose.yaml

```diff
@@ -33,7 +33,7 @@ services:
   # Sample web application which will generate Metrics and Traces data if the enable API is called
   # src - https://github.com/aws-observability/aws-otel-test-framework/tree/terraform/sample-apps/spark
   ot-sample-app:
-    image: aottestbed/aws-otel-collector-java-sample-app:0.9.0
+    image: public.ecr.aws/a3i2u7g3/aws-otel-test-framework-sample-apps-spark:0.10.0
```

Comment:
have you made any changes to the sample app code or just use the current code and republish it to ECR?

Reply:
Yes the test changes should be in the new ecr

Reply:
RUN wget https://github.com/aws-observability/aws-otel-java-instrumentation/releases/download/v1.4.1/aws-opentelemetry-agent.jar

Reply:
The changes should be the same as the ones in this PR: https://github.com/aws-observability/aws-otel-test-framework/pull/321